python/lsst/ts/standardscripts/auxtel/slew_telescope_icrs.py

```diff
@@ -82,14 +82,14 @@ def get_schema(cls):
                 type: number
                 minimum: -90
                 maximum: 90
-              rot_pa:
+              rot_sky:
```

Comment:
Do you want to support `rot_par` as well? If so you could simply omit the default for that. It will either show up as `None` or missing (I'm not sure which) in the config. If it's missing then it'd be easy enough to add it in the `config` method.

(Same question for the weirder one: rot_skyphys or whatever you decide to call it.)